pkg/ery/di/app.go

```diff
@@ -127,3 +137,28 @@ func (c *appComponentImpl) LocalDockerContainerRepository() domain.ContainerRepo
 	})
 	return c.localDockerContainerRepo
 }
+
+func (c *appComponentImpl) HTTPClient() *http.Client {
+	c.initHTTPClient.Do(func() {
+		dialerFunc := func(ctx context.Context, network, address string) (net.Conn, error) {
```

Comment:
**[unparam]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
(*appComponentImpl).HTTPClient$1$1 - network is unused